src/styles/App.sass

```diff
@@ -184,10 +184,15 @@ html
     margin-left: 1.5rem
 
 .project-card__image
-    width: 40%
-    height: 100%
+    width: 100%;
+    height: 100%;
+    
+
+.project-card__image-div
+    width: 400px;
+    height: 250px;
     margin-left: 0.7rem
-//     object-fit: contain
+    object-fit: contain
```

Comment:
This had made the images all stretch / shrink to fill the space so they all look whacky

![image](https://user-images.githubusercontent.com/16625180/67164304-8802fb80-f379-11e9-923a-cb8c1913b63d.png)